pkg/mockery/mockery.go

```diff
@@ -140,7 +142,14 @@ func (s MockHandler) RenderTemplateResponse(e Endpoint) (string, error) {
 					}
 					return w.Write([]byte(envVar))
 				}
-
+				if variable.Header != "" {
+					if h := header.Get(variable.Header); h != "" {
+						return w.Write([]byte(h))
+					}
+					// Headers are dynamic and only available in runtime, so fail silently if header is not set.
+					s.Log.Printf("HTTP header %s not found for template %s of endpoint %s", variable.Header, e.Template, e.Uri)
+					return 0, nil
```

Comment:
Should we return error 400 when request doesn’t contain expected headers?

Reply:
I think that it would break our `ValidateConfig()` logic, that is not aware of request headers.